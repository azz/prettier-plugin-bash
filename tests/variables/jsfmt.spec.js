run_spec(__dirname, ['bash']);
