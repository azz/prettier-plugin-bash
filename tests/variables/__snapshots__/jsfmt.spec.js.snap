// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`manipulation.sh 1`] = `
====================================options=====================================
parsers: ["bash"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
#!/bin/bash

foo=\${bar/1/2}
foo=\${bar//1/2}

foo=\${bar^}
foo=\${bar^^}

foo=\${bar,}
foo=\${bar,,}

=====================================output=====================================
foo=\${bar/1/2}
foo=\${bar//1/2}
foo=\${bar^}
foo=\${bar^^}
foo=\${bar,}
foo=\${bar,,}
================================================================================
`;

exports[`variables.sh 1`] = `
====================================options=====================================
parsers: ["bash"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
#!/bin/bash

FOO=bar

FOO=$(bar)

FOO=$((1))

declare -r FOO

=====================================output=====================================
FOO=bar
FOO=$(bar)
FOO=$((1))
declare -r FOO
================================================================================
`;
